site/src/helpers/courseRequirements.ts

```diff
@@ -160,7 +160,6 @@ export function flattenSingletonGroups(requirements: ProgramRequirement[]): Prog
  * @param requirements The program requirements list
  */
 export function sortGroupRequirementsByType(requirements: ProgramRequirement[]): ProgramRequirement[] {
-  // console.log(requirements)
```

Comment:
assuming this will be nuked when merged into main

Reply:
not sure what you mean since this is a line removal, but the commented console log will not be there

Reply:
oh i'm fukcign stupid

don't do code review on 4 hours of sleep kids